content/hardware/04.pro/boards/portenta-x8/datasheet/datasheet.md

```diff
@@ -559,8 +559,8 @@ Hereby, Arduino S.r.l. declares that this product is in compliance with essentia
 
 ## Change Log
 
-| **Date**   | **Changes**                            |
-|------------|----------------------------------------|
-| 26/01/2023 | Clarify open-source nature of M7 core  |
-| 12/09/2022 | Make cores clear, minor fixes          |
-| 24/03/2022 | Release                                |
+| **Date**   | **Revision** | **Changes**                            |
```

Comment:
Is there another word we can use here instead of "revision" or "version"? We're using that word so much already that I'm afraid to have it in yet another place